lib/clientv2/node.go

```diff
@@ -59,6 +62,63 @@ func (r nodes) Update(ctx context.Context, res *apiv2.Node, opts options.SetOpti
 
 // Delete takes name of the Node and deletes it. Returns an error if one occurs.
 func (r nodes) Delete(ctx context.Context, name string, opts options.DeleteOptions) (*apiv2.Node, error) {
+	pname, err := names.WorkloadEndpointIdentifiers{Node: name}.CalculateWorkloadEndpointName(true)
```

Comment:
I think the WEPs, IP and IPAMHost only need to be done for !KDD

Reply:
Isn't it up to the underlying backend driver to no-op on that?

Reply:
kinda - so if you want to go that route then you need to make sure you handle OperationNotSupported errors and just swallow them.

However, for KDD we don't do anything clever about prefix lists for WEPs, so you are basically gonna list all of the workload endpoints (Pods) and then do nothing with it.  Seems like a bit of an overkill.